### PR TITLE
Ddd note for bandwidth parameter

### PR DESCRIPTION
**Pull request by f-brinkmann**
### Which issue(s) are closed by this pull request?

Improves docs related to  #641 

Could also go into main, but I did not consider it a hotfix.

### CODE REVIEW

**Review by sikersten — DISMISSED** [review]

File: pyfar/dsp/filter/audiofilter.py [hunk 1]
```diff
@@ -232,6 +232,11 @@ def high_shelve_cascade(
 
     ``slope = gain/bandwidth``
 
+    .. note::
+        The `bandwidth` must be at least 1 octave to obtain a good
+        approximation of the desired frequency response. Make sure to specify
+        the parameters `gain`, `slope`, and `bandwidth` accordingly.
```
Comment (ahms5) [inline pyfar/dsp/filter/audiofilter.py:238]:
```suggestion
        The ``bandwidth`` must be at least 1 octave to obtain a good
        approximation of the desired frequency response. Make sure to specify
        the parameters ``gain``, ``slope``, and ``bandwidth`` accordingly.
```

File: pyfar/dsp/filter/audiofilter.py [hunk 2]
```diff
@@ -331,6 +336,12 @@ def low_shelve_cascade(
 
     ``slope = -gain/bandwidth``
 
+    .. note::
+
+        The `bandwidth` must be at least 1 octave to obtain a good
+        approximation of the desired frequency response. Make sure to specify
+        the parameters `gain`, `slope`, and `bandwidth` accordingly.
```
Comment (ahms5) [inline pyfar/dsp/filter/audiofilter.py:343]:
```suggestion
        The ``bandwidth`` must be at least 1 octave to obtain a good
        approximation of the desired frequency response. Make sure to specify
        the parameters ``gain``, ``slope``, and ``bandwidth`` accordingly.
```

File: pyfar/dsp/filter/audiofilter.py [hunk 3]
```diff
@@ -232,6 +232,11 @@ def high_shelve_cascade(
 
     ``slope = gain/bandwidth``
 
+    .. note::
```
Comment (ahms5) [inline pyfar/dsp/filter/audiofilter.py:235]:
```suggestion
    .. note::
    
```

**Review by ahms5 — CHANGES_REQUESTED** [review]
Thanks a lot for taking care. just small suggetions, maybe use `` instead of ` to make it as code in the doc.

File: pyfar/dsp/filter/audiofilter.py (hunk 1 above)
Comment (f-brinkmann) [reply pyfar/dsp/filter/audiofilter.py:238]: I considered it to be references for parameters, in which our guide lines suggest italic font.

**Review by ahms5 — APPROVED** [review]

**Review by ahms5 — APPROVED** [review]
I think it should go to main, since we dicided to merge to main, if its about documentation

**Comment by f-brinkmann** [comment]
closed in favor of #648

**Pull request closed without merging**